src/main/java/com/healthmarketscience/jackcess/util/MemFileChannel.java

```diff
@@ -116,13 +116,9 @@ public static MemFileChannel newChannel(File file) throws IOException {
   public static MemFileChannel newChannel(File file, String mode)
     throws IOException
   {
-    FileChannel in = null;
-    try {
-      return newChannel(in = new RandomAccessFile(
-                            file, RO_CHANNEL_MODE).getChannel(),
-                        mode);
-    } finally {
-      ByteUtil.closeQuietly(in);
+    try (RandomAccessFile raf = new RandomAccessFile(file, RO_CHANNEL_MODE);
```

Comment:
the original code was intentional.  i didn't want errors when closing the file channel to fail the method.

Reply:
I see.
Updated this part as follows. This is functionally equivalent but removes the Java warning:
```
{
  RandomAccessFile raf = null;
  FileChannel in = null;
  try {
    raf = new RandomAccessFile(file, RO_CHANNEL_MODE);
    return newChannel(in = raf.getChannel(), mode);
  } finally {
    ByteUtil.closeQuietly(in);
    ByteUtil.closeQuietly(raf);
  }
}

```